File: backend/app/models/comment.rb
class Comment < ApplicationRecord
    belongs_to :photo
    belongs_to :user
    
    validates :content, presence: true
    validates :user_id, presence: true
    def name=(name)
        if User.find_by(name: name)
            self.user = User.find_by(name: name)
        else
            User.create(name: name)
            self.user = User.find_by(name: name)
        end 
    end 
end

File: frontend/src/models/comment.js
class Comment {
    
    static all = []
    
    constructor(data){
        this.id = data.id
        this.content = data.content
        this.photo_id = data.photo_id
        this.user_id = data.user_id
        this.toggled = false
        this.save()
       
    }
    
    save() {
        Comment.all.push(this)
    }
    
    render_html() {
        if (this.toggled == false){
         let h6 = document.createElement("h6");
         console.log(User.all)
         let user = User.all.find(x => x.id === this.user_id)
         h6.innerText = `${user.name}:${this.content}`;
         return h6
         this.toggled = true
        } 
       
    }
}